src/transport.ts

```diff
@@ -24,4 +45,23 @@ export class WebRTCTransport implements Transport {
 	get [symbol](): true {
 		return true
 	}
+
+	todo_cb() {
+	}
+
+	_connect (ma: Multiaddr, options: WebRTCDialOptions) {
+		this.channel = this.peer_connection.createDataChannel("channel");
+		this.channel.onopen = this.todo_cb;
+		this.channel.onclose = this.todo_cb;
+		this.peer_connection.onicecandidate = this.todo_cb;
```

Comment:
This can be a noop as we have no direct way of exchanging candidates with the remote. The exchange happens via the SDP which is constructed by the server after looking at incoming STUN messages.